build/Dockerfile.cicd

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.15.2-buster
```

Comment:
AFAIK `Dockerfile` usually used as extension

Reply:
About GO version, I think we agreed on 1.14.4, at least that one used in the rest of pipeline.